src/Futurice/Time/Month.hs

```diff
@@ -167,6 +168,9 @@ instance ToHtml Month where
     toHtmlRaw = toHtml
     toHtml = toHtml . monthToText
 
+instance Csv.ToField Month where
```

Comment:
you should add `Csv.FromField` at the same time :)

Reply:
Good point, though that might be harder. :D